dashboard/db/migrate/20220603175343_code_review_remove_expires_at.rb

```diff
@@ -0,0 +1,5 @@
+class CodeReviewRemoveExpiresAt < ActiveRecord::Migration[6.0]
```

Comment:
Do you want to apply this separately, in case we need to revert the rest of the change?  (Up to you.)

Reply:
https://github.com/code-dot-org/code-dot-org/pull/46706